src/ExposureHistory/History/NoExposures.tsx

```diff
@@ -1,21 +1,97 @@
 import React, { FunctionComponent } from "react"
-import { View, StyleSheet } from "react-native"
+import env from "react-native-config"
+import { Linking, View, StyleSheet, TouchableOpacity } from "react-native"
 import { useTranslation } from "react-i18next"
 
 import { RTLEnabledText } from "../../components/RTLEnabledText"
 
 import { Colors, Typography, Spacing, Outlines } from "../../styles"
+import { SvgXml } from "react-native-svg"
+import { Icons } from "../../assets"
 
+const {
+  GAEN_AUTHORITY_NAME: healthAuthorityName,
+  AUTHORITY_ADVICE_URL: healthAuthorityLink,
+} = env
 const NoExposures: FunctionComponent = () => {
   const { t } = useTranslation()
   return (
-    <View style={style.container}>
-      <RTLEnabledText style={style.headerText}>
-        {t("exposure_history.no_exposure_reports")}
-      </RTLEnabledText>
-      <RTLEnabledText style={style.subheaderText}>
-        {t("exposure_history.no_exposure_reports_over_past")}
-      </RTLEnabledText>
+    <>
+      <View style={style.container}>
+        <RTLEnabledText style={style.headerText}>
+          {t("exposure_history.no_exposure_reports")}
+        </RTLEnabledText>
+        <RTLEnabledText style={style.subheaderText}>
+          {t("exposure_history.no_exposure_reports_over_past")}
+        </RTLEnabledText>
+      </View>
+      <View style={style.card}>
```

Comment:
what do you think about breaking this card into a subcomponent?  I would keep it in the same file, if so.